bpf/include/bpf_log.h

```diff
@@ -32,13 +32,31 @@
 #define BPF_LOGTYPE_SOCKOPS BPF_DEBUG_OFF
 #define BPF_LOGTYPE_XDP     BPF_DEBUG_OFF
 #define BPF_LOGTYPE_SENDMSG BPF_DEBUG_OFF
+#define MAX_MSG_LEN         255
+
 enum bpf_loglevel {
     BPF_LOG_ERR = 0,
     BPF_LOG_WARN,
     BPF_LOG_INFO,
     BPF_LOG_DEBUG,
 };
 
+struct log_event {
+    __u32 ret;
+    char msg[MAX_MSG_LEN];
+};
+struct {
+    __uint(type, BPF_MAP_TYPE_RINGBUF);
+    __uint(max_entries, 256 * 1024 /* 256 KB */);
+} kmesh_events SEC(".maps");
+
+struct {
+    __uint(type, BPF_MAP_TYPE_PERCPU_ARRAY);
+    __uint(max_entries, 1);
+    __type(key, int);
+    __type(value, struct log_event);
+} heap SEC(".maps");
```

Comment:
What's this map for? It's better to name it based on usage scenario

Reply:
> What's this map for? It's better to name it based on usage scenario

This map is to avoid exceeding ebpf stack which is 512B, so use this map to hold the log_event object.